### PR TITLE
Easy Insight integration

### PR DESCRIPTION
**Pull request by jboe**

### CODE REVIEW

File: integrations.yml [hunk 1]
```diff
@@ -40,3 +40,9 @@
   website: "https://hubstaff.com/time_tracking_for_basecamp_3"
   icon: "hubstaff.png"
   category: 2
+
```
Comment (javan) [inline integrations.yml]: Please add a `-` here instead of an empty line. For example:
```yaml
-
  name: "Easy Insight"
  ...
```

**Comment by javan** [comment]
Thanks! Does your integration work with Basecamp 3? It's not clear from viewing https://www.easy-insight.com/solutions/basecamp.html.

**Comment by jboe** [comment]
Updated the Basecamp page on our side to state that it works with Basecamp 3. Did the change to integrations.yml come through on this, or do I need to push something else from my side?

**Comment by javan** [comment]
Looks good, thanks!

**Pull request merged**